Ginger/GingerCoreNET/BPMN/ActivitiesGroupToBPMNConverter.cs

```diff
@@ -170,7 +223,7 @@ private bool IsWebServicesActivity(Activity activity)
                 .FirstOrDefault(platform => string.Equals(platform.AppName, activity.TargetApplication));
             if(activityAppPlatform == null)
             {
-                throw new InvalidOperationException($"No Application Platform found for Activity with Target Application '{activity.TargetApplication}'.");
+                throw new BPMNExportException($"No Application Platform found for Activity with Target Application '{activity.TargetApplication}'.");
```

Comment:
Use Ginger Resource dictionary to show entity names according to the selected terminology